### PR TITLE
[5.4] Add min_ratio/max_ratio to dimension validator

### PR DESCRIPTION
**Pull request by leo108**
We can now set the dimension ratio range to the image validator:

```
Rule::dimensions()->maxRatio(3)->minRatio(2);
```

or

```
dimensions:max_ratio=3,min_ratio=2
```

### CODE REVIEW

File: src/Illuminate/Validation/Concerns/ValidatesAttributes.php [hunk 1]
```diff
@@ -470,15 +470,33 @@ protected function failsBasicDimensionChecks($parameters, $width, $height)
      */
     protected function failsRatioCheck($parameters, $width, $height)
     {
-        if (! isset($parameters['ratio'])) {
+        if (! isset($parameters['ratio']) && ! isset($parameters['min_ratio']) && ! isset($parameters['max_ratio'])) {
```
Comment (tillkruss) [inline src/Illuminate/Validation/Concerns/ValidatesAttributes.php]: You can combine these into a single `isset()` statement.

**Review by tillkruss — CHANGES_REQUESTED** [review]

File: src/Illuminate/Validation/Concerns/ValidatesAttributes.php (hunk 1 above)
Comment (leo108) [inline src/Illuminate/Validation/Concerns/ValidatesAttributes.php]: > If multiple parameters are supplied then isset() will return TRUE only if all of the parameters are set

@tillkruss Is it possible to combine these into a single `isset()` statement?

Comment (tillkruss) [inline src/Illuminate/Validation/Concerns/ValidatesAttributes.php]: Ah, you're right.

**Review by tillkruss — APPROVED** [review]

**Comment by taylorotwell** [comment]
No plans to add this to the core.

**Pull request closed without merging**